manifests/y/Yandex/Browser/23.1.3.952/Yandex.Browser.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Yandex.Browser
+PackageVersion: 23.1.3.952

```

```suggestion
PackageVersion: 23.1.3.949
```
Must match Control Panel / Apps&Features